### ISSUE TITLE
Staging Review finding: Accordion headings are used incorrectly

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#90539]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vfep-codereviewers
**Product:** Resources for schools
**Feature:** Convert Legacy Markdown SCO page into a React App

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

The headings for accordions are all coded at the `h2` level. This does not accurately reflect the flow of the page and presents issues for screen reader users navigating the page by heading.

<details><summary>Link, screenshot or steps to recreate</summary>

![accordion-headers](https://github.com/user-attachments/assets/b668ad6e-7fef-4faf-a769-91fb56f293c2)

</details>

## Recommended action

The headings for accordions should be h3’s rather than h2's, as they are children of parent sections using h2's. You might consider not using accordions at all for this content, since all accordions are already auto-expanded on the page.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** [Accordion ](https://design.va.gov/components/accordion)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Page titles and section titles](https://design.va.gov/content-style-guide/page-titles-and-section-titles)
- **Context:** Headings

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by alexisbetz** [comment]
https://jira.devops.va.gov/browse/VEBT-417
